app/workers/hmrc_interface_bulk_submission_worker.rb

```diff
@@ -0,0 +1,22 @@
+class HmrcInterfaceBulkSubmissionWorker < ApplicationWorker
+  sidekiq_options queue: DefaultQueueNameService.call
+
+  # NOTE: ..to self on possible alternative to using sidekiq processes with concurrency of 1
```

Comment:
Remove comment and either use the alternative or create a ticket to use the alternative if required? (or possibly TODO?)